tests/regression/nextcloud-rule-exclusions-plugin/9508880.yaml

```diff
@@ -20,84 +20,10 @@ tests:
             uri: /ocs/v2.php/references/resolve
             data: "reference=https://www.jaycar.com.au/raspberry-pi-3-switchmode-power-supply-5-1v-2-5a-with-usb-micro-b/p/MP3536?pos=1&queryId=0a9ebf34f466c42c586d9561f4ac20d2&sort=relevance&searchText=2.5a%205v%20micro%20usb"
           output:
-            no_log_contains: id "920230"
-  - test_title: 9508880-2
```

Comment:
Why did you remove these tests?

Reply:
they all do the same thing, except they check for different rules. imo it looks cleaner this way since the test is testing for all the rules the payload triggers, instead of creating multiple tests for different rules.